manticore/core/state.py

```diff
@@ -412,9 +412,19 @@ def concretize(self, symbolic, policy, maxcount=7):
                 vals = (True,)
         else:
             assert policy == "ALL"
-            vals = self._solver.get_all_values(
-                self._constraints, symbolic, maxcnt=maxcount, silent=True
-            )
+            if additional_symbolics is not None:
+                print(symbolic, additional_symbolics)
```

Comment:
If you do end up leaving `additional_symbolics`, this should be swapped out for `logger.debug` or `logger.info`

Reply:
Fixed.